src/gatling/simulations/uk/gov/hmcts/reform/sandl/snlrules/performance/NewSessionSimulation.scala

```diff
@@ -0,0 +1,37 @@
+package uk.gov.hmcts.reform.sandl.snlrules.performance
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+import scala.concurrent.duration._
+
+class NewSessionSimulation extends Simulation {
+  val httpConf = http
+    .baseURL(Environments.baseUrl)
```

Comment:
Base url seems to be not using ignore certificates like "acceptAnyCertificate = true  "  will that work is the PR does not have valid certificate?

Reply:
uncommented the option in conf file and set to true